server/forms.py

```diff
@@ -473,7 +473,23 @@ class GithubSearchRecentForm(BaseForm):
     issue_title = StringField('Issue Title (Optional)', validators=[validators.optional()],
                                 default="Academic Integrity - Please Delete This Repository")
     issue_body = TextAreaField('Issue Body (Optional)', validators=[validators.optional()],
-                               description="The strings '{repo}' and '{author}' will be replace with the approriate value")
+                               description="The strings '{repo}' and '{author}' will be replace with the appropriate value")
+
+class EmailScoresForm(BaseForm):
+    subject = StringField('Subject', validators=[validators.required()])
+    body = TextAreaField('Body', validators=[validators.required()],
+                       description="Is there anything the students need to know about the scores?",
+                       default="An instructor has published scores. If you have questions, please contact the course staff")
+    reply_to = EmailField('Reply To Address', default="no-reply@okpy.org",
```

Comment:
From always has to be an okpy.org domain to send reliably - we can't (reliably) spoof the user's email address. 

When someone hits reply their mail client uses the reply-to address if available (and if it's not it uses the "From" address). 

The only thing I'm unsatisfied with in this PR is that we still use the no-reply email even if there is a valid reply to address (making it seem like the user can't reply). I could generate an email for the course I guess?